website/pages/en/cookbook/arweave.mdx

```diff
@@ -2,7 +2,7 @@
 title: Building Subgraphs on Arweave
 ---
 
-> Arweave support in Graph Node and on the hosted service is in beta: please reach us on [Discord](https://discord.gg/graphprotocol) with any questions about building Arweave subgraphs!
+> Arweave support in Graph Node and on the Subgraph Studio is in beta: please reach us on [Discord](https://discord.gg/graphprotocol) with any questions about building Arweave subgraphs!
```

Comment:
Do we say "the" Subgraph Studio or just on (or in?) Subgraph Studio?

Reply:
I think maybe just "in", good call